### ISSUE TITLE
Create a worker to check balances of the sponsor wallets that need to be topped up

### ISSUE THREAD

**Issue by metobom** [api3dao/wallet-watcher#30]
> A different worker (wallet-watcher probably) must check balances of the sponsor wallets that need to be topped up. It will also probably need to check balances of FunderDepository contracts and account used to send Funder.fund() transactions.

**Comment by acenolaza** [comment]
This is basically a duplicate of https://github.com/api3dao/operations-database/issues/285 and has been unblocked by the refactor made to wallet-watcher [here](https://github.com/api3dao/wallet-watcher/pull/29)
